lib/style-transform.js

```diff
@@ -1,407 +1,126 @@
-// based on Stylis (MIT)
-// modified by Guillermo Rauch to add support for custom
-// attributes instead of custom selector prefix
-/*!
- *
- *          __        ___
- *    _____/ /___  __/ (_)____
- *   / ___/ __/ / / / / / ___/
- *  (__  ) /_/ /_/ / / (__  )
- * /____/\__/\__, /_/_/____/
- *          /____/
- *
- * stylis is a small css compiler
- *
- * @licence MIT
- */
-(function (factory) {
-    if (typeof exports === 'object' && typeof module !== 'undefined') {
-        module.exports = factory(global);
-    } else if (typeof define === 'function' && define.amd) {
-        define(factory(window));
-    } else {
-        window.stylis = factory(window);
+const stylis = require('stylis')
+
+function middleware(ctx, str, line, col, prefix) {
+  // ctx 1.5 is a selector
+  if (ctx !== 1.5) {
+    return
+  }
+
+  // Stylis adds the namespace at the beginning
+  if (str.indexOf(prefix + ' ') !== 0) {
+    return
+  }
+
+  // while want to append it to selectors.
+  // Strip stylis' namespace.
+  str = str.substring(prefix.length).trim();
+
+  var i = 0
```

Comment:
Ok, will test it out locally to see what tests fail, might be vendor prefixing and whitespace issues since stylis has gone through a few patches since.